### PR TITLE
Cleaned up event subscriber

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Cleaned up event subscriber. Theres no need to extend an existing event subscriber for the things this is doing.

This should be able to be merged independently of https://github.com/SU-SWS/nobots/pull/4

### CODE REVIEW

File: src/EventSubscriber/FinishResponseSubscriber.php [hunk 1]
```diff
@@ -1,26 +1,31 @@
...
 
   /**
    * Sets extra headers on successful responses.
    *
    * @param \Symfony\Component\HttpKernel\Event\FilterResponseEvent $event
    *   The event to process.
    */
-  public function onRespond(\Symfony\Component\HttpKernel\Event\FilterResponseEvent $event) {
+  public function onRespond(FilterResponseEvent $event) {
     $request = $event->getRequest();
```
Comment (sherakama) [inline src/EventSubscriber/FinishResponseSubscriber.php]: Probably can remove this line as well as it is not used.

**Review by sherakama — APPROVED** [review]

File: src/EventSubscriber/FinishResponseSubscriber.php (hunk 1 above)
Comment (pookmish) [reply src/EventSubscriber/FinishResponseSubscriber.php]: That's in the other PR. I didn't wanna create conflicts

Comment (sherakama) [reply src/EventSubscriber/FinishResponseSubscriber.php]: Just saw that. Ignore.

Comment (pookmish) [reply src/EventSubscriber/FinishResponseSubscriber.php]: Dang. Conflicts still came up.

**Review by sherakama — APPROVED** [review]

**Pull request merged**